.github/scripts/test_kilosort4_ci.py

```diff
@@ -554,16 +562,18 @@ def _get_kilosort_native_settings(self, recording, paths, param_key, param_value
             "n_chan_bin": recording.get_num_channels(),
             "fs": recording.get_sampling_frequency(),
         }
+        run_kilosort_kwargs = {}
 
-        if param_key == "binning_depth":
-            settings.update({"nblocks": 5})
+        if param_key is not None:
+            if param_key == "binning_depth":
+                settings.update({"nblocks": 5})
 
-        if param_key in RUN_KILOSORT_ARGS:
-            run_kilosort_kwargs = {param_key: param_value}
-        else:
-            if param_key != "change_nothing":
-                settings.update({param_key: param_value})
-            run_kilosort_kwargs = {}
+            if param_key in RUN_KILOSORT_ARGS:
+                run_kilosort_kwargs = {param_key: param_value}
+            else:
+                if param_key != "change_nothing":
```

Comment:
was this removed? so should be `if param_key`?

Reply:
good catch!

Reply:
done :)